hudi-client/hudi-spark-client/src/test/java/org/apache/hudi/testutils/HoodieClientTestHarness.java

```diff
@@ -112,10 +110,9 @@
 /**
  * The test harness for resource initialization and cleanup.
  */
-public abstract class HoodieClientTestHarness extends HoodieCommonTestHarness {
+public abstract class HoodieClientTestHarness extends HoodieWriterClientTestHarness {
```

Comment:
why can't we use HoodieClientTestHarness as the base instead of creating yet another test harness. other test classes extending from HoodieWriterClientTestHarness can extend from HoodieClientTestHarness.

Reply:
Are you suggesting removing the spark specific code from HoodieClientTestHarness or making all client modules depend on spark?

Reply:
I see. lets rename HoodieClientTestHarness to HoodieSparkClientTestHarness then